### PR TITLE
21.1.5 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: CS/ContextMenuCustomizations_Ex/Window1.xaml [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
-                <dxb:BarButtonItem Name="bAbout" CategoryName="Help" Content="About" ItemClick="bAbout_ItemClick" dxb:BarItemLinkActionBase.ItemLinkIndex="0" />
-                <!--Add a separator-->
+                <!--  Adds a new About menu command at the top  -->
+                <dxb:BarButtonItem
+                    Name="bAbout"
+                    dxb:BarItemLinkActionBase.ItemLinkIndex="0"
+                    CategoryName="Help"
+                    Content="About"
+                    ItemClick="bAbout_ItemClick" />
+                <!--  Add a separator  -->
```
Comment (albertov05) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]: Should this comment be "Adds a separator"?

File: CS/ContextMenuCustomizations_Ex/Window1.xaml [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
                 <dxb:InsertBarItemLinkAction ItemLinkIndex="1">
                     <dxb:BarItemLinkSeparator />
                 </dxb:InsertBarItemLinkAction>
-            <!--Remove a comannd from the context menu-->
-            <dxb:RemoveAction ElementName="{x:Static dxdo:DefaultMenuItemNames.ClosedPanels}"/>
-            <!--Change the Hide button's caption -->
-            <dxb:UpdateAction ElementName="{x:Static dxdo:DefaultMenuItemNames.Hide}" PropertyName="Content" Value="Hide Item"/>
+                <!--  Remove a comannd from the context menu  -->
```
Comment (albertov05) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]: Note that "commannd" is misspelled.

Also, should "Remove" be "Removes"?

File: CS/ContextMenuCustomizations_Ex/Window1.xaml [hunk 3]
```diff
@@ -1,22 +1,45 @@
...
+                <dxb:RemoveAction ElementName="{x:Static dxdo:DefaultMenuItemNames.ClosedPanels}" />
+                <!--  Change the Hide button's caption  -->
```
Comment (albertov05) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]: Should "Change" be "Changes"?

File: Readme.md [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
-
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1631)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager -  Use Actions to Customize a Panels Context Menu
```
Comment (albertov05) [inline Readme.md]:
```suggestion
# WPF Dock Layout Manager - Use Actions to Customize a Panel's Context Menu

```

File: Readme.md [hunk 5]
```diff
@@ -1,19 +1,30 @@
...
+- [Dock UI Items](https://docs.devexpress.com/WPF/7209/controls-and-libraries/layout-management/dock-windows/dock-items)
+- [Customization Actions](https://docs.devexpress.com/WPF/7045/controls-and-libraries/ribbon-bars-and-menu/bars/bar-actions)
+- [BarButtonItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarButtonItem)
+
+## More Examples
+
+- [WPF Dock Layout Manager - Create a Simple Layout of Dock Panes](https://github.com/DevExpress-Examples/how-to-create-a-simple-layout-of-dock-panes-e1600)
+- [WPF Dock Layout Manager - Create a Complex Layout of Dock Panels](https://github.com/DevExpress-Examples/how-to-create-a-complex-layout-of-dock-panels-e1663)
+- [WPF Dock Layout Manager - Сreate a DocumentGroup with Two Tabs](https://github.com/DevExpress-Examples/how-to-create-a-documentgroup-with-two-tabs-e1670)
+- [WPF Dock Layout Manager - Create a Tabbed and Document Groups](https://github.com/DevExpress-Examples/how-to-create-a-tabbedgroup-and-documentgroup-groups-e1656)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
- [WPF Dock Layout Manager - Create Tabbed and Document Groups](https://github.com/DevExpress-Examples/how-to-create-a-tabbedgroup-and-documentgroup-groups-e1656)

```

File: CS/ContextMenuCustomizations_Ex/Window1.xaml (hunk 1 above)
Comment (friedcucumber) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]:
```suggestion
                <!--  Adds a separator  -->
```

File: CS/ContextMenuCustomizations_Ex/Window1.xaml (hunk 2 above)
Comment (friedcucumber) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]:
```suggestion
                <!--  Removes a command from the context menu  -->
```

File: CS/ContextMenuCustomizations_Ex/Window1.xaml (hunk 3 above)
Comment (friedcucumber) [inline CS/ContextMenuCustomizations_Ex/Window1.xaml]:
```suggestion
                <!--  Changes the Hide button's caption  -->
```

**Pull request merged**